.github/workflows/SHA-version-update.yaml

```
@@ -0,0 +1,12 @@
+name: SHA version update
+
+on:
+  pull_request:
+    branches: [ master ]
+    paths: ['dependabot/pip/dot-github/dependabot/*']
```

I manually rebased this onto a recent dependabot created PR: https://github.com/panosc-eu/spack-repo/pull/30

Noticed that it did not run, as the condition was:

```yaml
on:
  pull_request:
    branches: [ master ]
    paths: ['dependabot/pip/dot-github/dependabot/*']
```

I think that's from me telling you the wrong thing :stuck_out_tongue: that means that it will execute on a, if a file matching the pattern in `paths` has been modified as part of the PR. This should be something like:

```suggestion
  pull_request:
    branches: ['dependabot/pip/dot-github/dependabot/*']
```

---

in your example you also triggered on push https://github.com/panosc-eu/spack-repo/pull/30/files wouldn't that fail if the commit messages are not the original ones from dependabot? I' ve added an if statement now to check if the commit message contains Bump...from...to